.github/workflows/linux.yml

```diff
@@ -940,13 +959,28 @@ jobs:
     defaults:
       run:
         shell: bash
-    runs-on: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-8-cores' || 'ubuntu-20.04'}}
+    runs-on: ${{ github.event_name == 'schedule' && 'aks-linux-16-cores' || 'aks-linux-8-cores'}}
```

Comment:
does it benefit 4x from 8 cores?
I see the execution time improved from 24 to 14 mins. Maybe 4 cores are enough?

Reply:
4x is broken for this flow due to lack of the ram memory (
@ababushk  when we can reconfigure 4x runner?